packages/react-native-bpk-theming/stories.js

```diff
@@ -45,15 +44,6 @@ type Theme = {
   buttonSecondaryTextColor: string,
   buttonSecondaryBackgroundColor: string,
   buttonSecondaryBorderColor: string,
-  primaryColor: string,
-  colorGray50: string,
-  colorSkyGrayTint06: string,
-  colorSkyGrayTint05: string,
-  colorSkyGrayTint04: string,
-  colorSkyGrayTint03: string,
-  colorSkyGrayTint02: string,
-  colorSkyGrayTint01: string,
-  colorSkyGray: string,
```

Comment:
This was all for the brand refresh and it's no longer needed, or even supported? Idk but let's get rid of it